tests/test_amazon_sqs_block.py

```diff
@@ -14,10 +14,10 @@ def test_send_message(self):
         """Signals ..."""
         blk = SQSSendMessage()
         self.configure_block(blk, {
-            "region_name": "{{ $region_name }}",
-            "aws_access_key_id": "{{ $aws_access_key_id }}",
-            "aws_secret_access_key": "{{ $aws_secret_access_key }}",
-            "queue_url": "{{ $queue_url }}",
+            "region_name": "us-west-1",
```

Comment:
Aren't these props inside the `creds` prop? It doesn't look like these are doing anything like this.